src/app/services/auth.service.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { AuthService } from './auth.service';
+import { TestBed, inject } from '@angular/core/testing';
+import { OAuthService } from 'angular-oauth2-oidc';
+
+describe('AuthService', () => {
+  const fakeToken = {};
+  const fakeRoles = ['Administrators'];
+  const fakeRoleWithoutUsers = ['Foo'];
+
+  beforeEach(() => {
+     TestBed.configureTestingModule({
+       providers: [
+        AuthService,
+        {provide: OAuthService, useValue: {
+          getIdentityClaims: () => ['Administrators']
+        }}
+       ]
+     });
+  });
+
+  it('should be created', inject([AuthService], (service: AuthService) => {
+    expect(service).toBeTruthy();
+  }));
+
+  it('should return true given a valid auth token',
+    inject([AuthService], (service: AuthService) => {
+    const spy = spyOn(service, 'loggedIn').and.returnValue(!!fakeToken);
+    const rvl = service.loggedIn();
+    expect(spy).toHaveBeenCalled();
```

Comment:
don't need to check this if we're checking that it's called with nothing next

Reply:
Alright.